core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -200,7 +201,7 @@ private String getUserFromSignature() throws S3Exception {
    * @return the user
    */
   @VisibleForTesting
-  public static String getUserFromAuthorization(String authorization) {
+  public static String getUserFromAuthorization(String authorization) throws S3Exception {
     if (authorization == null) {
       return null;
     }
```

Comment:
After some discussion I agree that the "Authorization" header shouldn't be optional, so we can return the same `AuthorizationHeaderMalformed` error code here.

Reply:
done